packages/react/src/select/root/SelectRoot.test.tsx

```diff
@@ -2052,7 +2052,7 @@ describe('<Select.Root />', () => {
     it('falls back to null when both selected and initial default are removed (controlled)', async () => {
       function TestControlled() {
         const [items, setItems] = React.useState(['a', 'b', 'c']);
-        const [value, setValue] = React.useState('c');
+        const [value, setValue] = React.useState<string | null>('c');
```

Comment:
Without overloads, you're now forced to type `<string | null>` in state hook calls for controlled usage